src/types/allBooks.ts

```diff
@@ -0,0 +1,5 @@
+import BookObject from "./book";
+type allBooks = {
+  books: BookObject[];
+};
+export default allBooks;
```

Comment:
named exports are generally better than default exports and you should try to consistently always use them and don't use the default ones. (better for refactoring)